### ISSUE TITLE
⛔ 🛟 Support Request: Mobile background hover feature unresponsive 

### ISSUE THREAD

**Issue by Thabang28** [elementor/elementor#16383]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest development version of Elementor.

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro) with development versions.
- [ ] This bug happens with a vanilla WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [ ] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```
</details>

**Comment by Thabang28** [comment]
Hi github

I've got a problem related to a
Mobile elementor pro feature
Background hover image for mobile is not responsive on live tests when pressing the menu as it's supposed to.. it shows a white background instead of the picture which is 795KB. Jpg
It worked previously then I updated the pro version and it hasn't worked since

please help

![Screenshot_20210922_120951](https://user-images.githubusercontent.com/91189166/134325959-583d0ab8-e131-44df-a691-0e3743e7a213.jpg)

Everythingwomen4women.com

**Comment by rami-elementor** [comment]
@Thabang28 is this issue still relevant wit the latest Elementor & Elementor Pro versions?

**Comment by Thabang28** [comment]
@rami-elementor No, my issue hasnt been resolved after months of waiting. My elementor pro has been updated yet still the feature ive selected for mobile in my theme builder still shows a white background instead of the image i chose. Who should i contact to assist me sir Rami?

**Comment by rami-elementor** [comment]
Hi @Thabang28,

It sounds like a support question, not a bug report.

Please note that GitHub issues are reserved for **bug reports** and **feature requests**, they are **not used for support**.

For support, feel free to check out:

* [Elementor Community](https://www.facebook.com/groups/Elementors/) on Facebook
* [Elementor Knowledge base](https://docs.elementor.com)
* [Support forums](https://wordpress.org/support/plugin/elementor) on WordPress.org 
* [Elementor Developers docs](https://developers.elementor.com/)

If you’ve purchased [Elementor Pro](https://elementor.com/pro), you are entitled to either premier or VIP support. To open a ticket, go to your [Elementor Account page](https://my.elementor.com/) and click on the support icon for details.

Regards,

**Comment by rami-elementor** [comment]
Also, please provide steps to reproduce the issue.
